examples/hotrod/README.md

```diff
@@ -44,3 +44,29 @@ Then open http://127.0.0.1:8080
 
 [hotrod-tutorial]: https://medium.com/@YuriShkuro/take-opentracing-for-a-hotrod-ride-f6e3141f7941
 [hotrod-openshift]: https://blog.openshift.com/openshift-commons-briefing-82-distributed-tracing-with-jaeger-prometheus-on-kubernetes/
+
+
+
+# HotROD on Containerized Version
+## Requirements
```

Comment:
this is a bit confusing - the point is to let people run from published containers, is it not? In which case `go` is not a requirement. I think the instruction on how to run should be around L34. Basically think about it holistically - if someone who doesn't even know Go but wants to play with hotrod/jaeger, we should have instructions right at the top, and then below "if you want to build from source then ... (L34-39)"

Reply:
I would remove everything and just keep two `docker run` commands :)